src/cheri-pte-ext.adoc

```diff
@@ -141,11 +141,18 @@ bit value of the capability read. This will introduce additional traps during re
 |===
 |PTE.CW |PTE.CRG |Store/AMO
 | 0     | 0      | Page fault if stored tag is set
-| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^1^
+| 0     | 1      | Page fault if stored tag is set, or hardware CW and CRG update^2^
 | 1     | X      | Normal operation
 |===
 
-^1^ The choice here follows the pattern of whether to implement the _Svade_ extension
+^1^ The choice here is whether to take a data dependent exceptions on loads or atomic operations.
+ If the exception is taken then the software will sweep the page and update the generation
+ (CRG). If the exception is taken without checking the tag, then the sweep will happen sooner.
```

Comment:
I think that not checking the tag in a data-dependent way for loads is likely to cause more (unnecessary) traps which might degrade performance a little. It might also mean that pages are swept sooner (?), but I don't know if that is true, and you can probably write your sweeper to avoid that situation.

Reply:
text removed